config/locales/en/course/assessment/question/text_response.yml

```diff
@@ -4,14 +4,15 @@ en:
       question:
         text_responses:
           new:
-            header: 'New Text Response Question'
+            text_response: 'New Text Response Question'
+            file_upload: 'New File Upload Question'
           create:
-            success: 'The text response question was created.'
+            success: 'The %{name} was created.'
```

Comment:
The name here is `file upload question` or `text response question`. `display_title` is not inputted here.